README.md

```diff
@@ -181,30 +181,6 @@ sbot.db.query(
 )
 ```
 
-#### About-self
```

Comment:
I've removed this section of the README so people don't use this API here.
Alternatiively we could leave the heading and say:

```
There used to be a plugin hosted by this module. Please use `ssb-about-self` which now provides this functionality
```

Reply:
Removing seems fine IMO